SIPs/sip-1.md

```diff
@@ -0,0 +1,85 @@
+---
+sip: 1
+title: SIP Process and Guidelines
+author: Team TSS
+discussions-to: N/A
+status: Draft
+category: Informational
+created: 2018-Jun-4
+maintainer: Team TSS
+---
+
+# SIP-1: SIP Process and Guidelines
+
+## Abstract
+
+This document establishes the Symbol Improvement Proposal (SIP) process. It defines how proposals are created, reviewed, and accepted, and includes roles, categories, status flows, and templates to guide authors, maintainers, and the broader community.
+
+## Current
+
+Currently, there is no standardized process for proposing changes to Symbol. As a result, community members often find it difficult to contribute ideas or understand how proposals are reviewed and adopted.
+
+## Suggested
+
+- Establish a standardized SIP process and markdown template.
+- Define clear roles and responsibilities for authors and maintainers.
+- Use GitHub as the central hub for SIP tracking and discussions.
+
+## Rationale
+
+The Symbol community regularly generates valuable ideas, but many contributors are unsure how to formally submit or advocate for changes. Discussions often happen in informal channels like Discord or Twitter and are quickly lost. This proposal provides a clear, transparent, and collaborative pathway for meaningful contributions to Symbol's evolution.
+
+
+## Implementation
+
+### SIP Workflow
+
+```mermaid
+sequenceDiagram
+participant Proposer
+participant Discord/Twitter
+participant Maintainer
+participant GitHub
+
+Proposer->>Discord/Twitter: Discuss idea with the community
```

Comment:
Why don't you add the numbers below to the arrows, so it's even easier to follow the process?

Reply:
sound good to me 👍🏼